content/en/contribute/code/core/dev-environment.md

```diff
@@ -137,6 +139,45 @@ Now you can start CouchDB. The login for your CHT instance will be `medic` and t
 cd ~/cht-docker 
 COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose -f docker-compose.yml -f couchdb-override.yml up -d
 ```
+#### CouchDB Setup in 4.x for Apple Silicon
+
+You will need to build the Docker images locally on your machine because the images hosted publicly do not currently support Apple's ARM architecture. While in the root directory of your `cht-core` project:
+
+```shell
+npm run local-images
+```
+
+After the `npm` command completes successfully, a `local-build` folder will be created in the root directory of your `cht` project.
+
+Open the project with your favorite text editor, navigate inside the `local-build` folder and update the `cht-couchdb.yml` file with the necessary ports. 
+
+In the `couchdb` configuration inside the `cht-couchdb.yml` file, right after the `volumes` property, add the following properties and save the changes: 
+
+```
+ports:
+  - "5984:5984"
+  - "5986:5986"
+expose:
+  - "5984"
+  - "5986"
+```
+
+Set environment variables required by `npm` and `node`
+
+```shell
+echo "export COUCH_NODE_NAME=nonode@nohost">> ~/.$(basename $SHELL)rc
+echo "export COUCH_URL=http://medic:password@localhost:5984/medic">> ~/.$(basename $SHELL)rc
+. ~/.$(basename $SHELL)rc
+echo "export COUCH_DB_USER=medic">> ~/.$(basename $SHELL)rc
+echo "export COUCH_DB_PASSWORD=password">> ~/.$(basename $SHELL)rc
+```
+
+In your terminal, navigate inside the `local-build` folder and start the Docker containers:
```

Comment:
can we try to kill all running containers before starting this one?
`docker` kill $(docker ps -q)`